README.md

```diff
@@ -28,6 +28,13 @@ lc ci
 
 ## Getting Started
 
+### Prerequisites
+
+Please do not install latest version of docker and vagrant. These version are currently supported:
+
+* Docker 1.9.1 is latest supported version
+* Vagrant 1.8.5 is latest supported version
```

Comment:
`lc` doesn't use Vagrant. The reason why we limit Vagrant to 1.8.5 internally is because it has a problem with assigning a static IP, when Docker is in use. Someone who is just using `lc` by itself can use whatever version they want.

As for Docker itself, we now support up to 1.11.2.